src/components/People/People.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from 'react';
+import { PeopleType } from '../../Type/People';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+const peopleList = (peoples: PeopleType[]) => {
+  const newPeople = peoples.map((person) => {
+    const mother = peoples.find((p) => p.name === person.motherName);
+    const father = peoples.find((p) => p.name === person.fatherName);
+
+    return {
+      ...person,
+      mother: mother || null,
+      father: father || null,
+      motherName: person.motherName || '-',
+      fatherName: person.fatherName || '-',
+    };
+  });
+
+  return newPeople;
+};
+
+export const People = () => {
+  const [people, setPeople] = React.useState<PeopleType[]>([]);
+  const [isError, setIsError] = React.useState<boolean>(false);
+  const [isLoading, setIsLoading] = React.useState<boolean>(true);
```

Comment:
It's extra, bc ts already knows about this
```suggestion
  const [isError, setIsError] = useState(false);
  const [isLoading, setIsLoading] = useState(true);
```